contents/handbook/company/team/paolo-damico.md

```diff
@@ -4,8 +4,26 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Q1 2022 Goals
+- **Product strategy.**
+  - 2x ambitious Product milestones (like Experimentation). Strategy >> Execution. Exciting/evangelism.
```

Comment:
I guess this is a bit of a placeholder - but would be great if you could combine this into a single statement

Reply:
Would love to weave in totally nailing the "why component" of every milestone with data, customer and industry context.